docs/resource_hooks.md

```diff
@@ -59,3 +59,12 @@ The following policies define when the hook will be deleted.
 |--------|-------------|
 | `HookSucceeded` | The hook resource is deleted after the hook succeeded (e.g. Job/Workflow completed successfully). |
 | `HookFailed` | The hook resource is deleted after the hook failed. |
+
+As an alternative to hook deletion policies, both Jobs and Argo Workflows support the
+[`ttlSecondsAfterFinished`](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/)
```

Comment:
Seems like many users forget to delete workflows.
Should we set ttlSecondsAfterFinished to a long default value?

Reply:
For Argo CD v0.11, they should not forget because hooks are now visible from UI. We shouldn't choose for them.